src/core/time.d

```diff
@@ -4714,34 +4719,29 @@ double _abs(double val) @safe pure nothrow @nogc
   +/
 string numToString(long value) @safe pure nothrow
```

Comment:
Sigh. Yet another number-to-string conversion function. But fixing this should be a separate PR.